### DISCUSSION TITLE
Lammps can't recognize the fix style “plumed”

### DISCUSSION THREAD

**Discussion by songjia11** [deepmodeling/deepmd-kit#2995] [Q&A]
1. The deepmdkit-v2.2.7, plumed-v2.9.0, lammps-v2023.8.2.1.0, and dpgen-v0.12.0 are installed as follows,
conda create -n deepmd
conda activate deepmd
pip install deepmd-kit[gpu,cu11,lmp,ipi]
pip install dpgen
pip install plumed
2. the content of input.lammps
#Run MD - equil at 300K
run_style      verlet #Velocity verlet
fix            dpgen_plm all plumed plumedfile input.plumed outfile output.plumed
fix            1 all nvt temp ${TEMP} ${TEMP} 0.1 #NH thermostat - 300K with 100 fs frequency
fix            2 all momentum 1 linear 0 0 0 #Remove total linear momentum of the system at each step
fix            3 all recenter INIT INIT INIT
thermo_style   custom step temp pe etotal press #Setting printing
thermo         ${THERMO_FREQ} #Ouputing thermodynamic properties
dump            dpgen_dump all custom 10 traj/*.lammpstrj id type x y z
#dump          2 all custom 100 vel.xyz id type vx vy vz
run            ${NSTEPS} #25 ps
3. the error is
ERROR: Unrecognized fix style 'plumed' is part of the PLUMED package which is not enabled in this LAMMPS binary. (src/src/modify.cpp:917)
Last command: fix            dpgen_plm all plumed plumedfile input.plumed outfile output.plumed

**Comment by njzjz** [comment]
The LAMMPS binary we built for pip does not enable PLUMED. The problem here is that the "plumed" package on PyPI contains a Python wrapper without any binary.

**Reply by songjia11** [reply]
Thanks for your reply. I used Conda to install Deepmdkit, but I can only install version 2.0.3. Using plumed (2.8.0), the following error will occur, 
ERROR: Incompatible API version for PLUMED in fix plumed. Only Plumed 2.4.x, 2.5.x, 2.6.x, 2.7.x are tested and supported. (src/PLUMED/fix_plumed.cpp:81)
Last command: fix            dpgen_plm all plumed plumedfile input.plumed outfile output.plumed

One method is to install other versions of deepmdkit using Conda, and I also saw what you mentioned about modifying the environment variable, export Conda_ OverRIDE_ GLIBC="2.17" and export CONDA_ OverRIDE_ CUDA="11.6", but there are still mismatched errors like this "[#1572](https://github.com/deepmodeling/deepmd-kit/issues/1572)".
Please help me solve this problem.
